Apps/phradar/radar.json

```diff
@@ -0,0 +1,550 @@
+                "Pacific/Palau",
+                "Pacific/Pitcairn",
+                "Pacific/Pohnpei",
+                "Pacific/Port_Moresby",
+                "Pacific/Rarotonga",
+                "Pacific/Tahiti",
+                "Pacific/Tarawa",
+                "Pacific/Tongatapu",
+                "Pacific/Wake",
+                "Pacific/Wallis",
+                "UTC"
+            ],
+            "order": 3
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate Radar asset configuration for connectivity using supplied configuration",
+            "verbose": "",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create privacy incident",
+            "identifier": "create_privacy_incident",
+            "description": "Create a Radar incident.",
+            "verbose": "<p>This action creates a privacy incident in Radar using the following phantom system and app configurations:</p>\n<h4>Company Settings</h4>\n<p><b>Base URL For Phantom Appliance</b> - used to link the created incident's 'intake method' to this instance of Phantom.</p>\n\n<h4>Asset Configuration</h4>\n<p>Asset configuration values are used as follows to create the incident:</p>\n<p><b>Incident Group ID</b> - must be specified in the asset configuration for authorization.</p>\n<p><b>Time Zone</b> - the current moment that this action is triggered is used to set the created incident's 'discovered' date and time. The asset <b>Time Zone</b> is used to specify the 'discovered' time zone offset.</p>\n\n\n<h4>Container Information</h4>\n<p>The container ID for the associated phantom event or case is use to specify the Radar incident channel ID. This is used to associate the created incident to the container</p>\n\n<h4>Action Parameters</h4>\n<p>The action parameters are used to set the Radar incident <b>name</b> and <b>description</b></p>\n",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "name": {
+                    "description": "A name summarizing the privacy incident.",
+                    "data_type": "string",
+                    "required": true,
+                    "default": "",
+                    "order": 0
+                },
+                "description": {
+                    "description": "A description of the privacy incident (plain text only).",
+                    "data_type": "string",
+                    "order": 1
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.data.*.incident_id",
+                    "data_type": "string",
+                    "column_name": "incident id",
+                    "column_order": 0
```

Comment:
If you add the key `contains`, with value as `[ "radar incident id" ]` then wherever there is incident_id and the contains mentioned, there will be contextual actions implemented courtesy of the platform